ee/app/livechat-enterprise/server/settings.js

```diff
@@ -115,6 +63,93 @@ export const createSettings = () => {
 				],
 			});
 		});
+
+		this.section('Queue_management', function() {
```

Comment:
```suggestion
		this.section('Queue_Management', function() {
```

Otherwise, translation added [here](https://github.com/RocketChat/Rocket.Chat/pull/22955/files#diff-df6fd124535e1dc52dbd93a30d84f6be0b9b65083f81233b80f30878cfbf9d89R3356) won't be applied

Reply:
Fixed in next commit